File: Java/src/test/java/com/pduda/field/ImagineThisIsAClientInADifferentRepositoryTest.java
package com.pduda.field;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class ImagineThisIsAClientInADifferentRepositoryTest {


    @Test
    public void singleItem_numberOfProductsInTheCart() throws Exception {
        ImagineThisIsAClientInADifferentRepository client = new ImagineThisIsAClientInADifferentRepository();

        assertThat(client.formattedTotalPrice(50)).isEqualTo("Total price is 50 euro");
    }

}

File: Kotlin/src/test/kotlin/com/pduda/field/ImagineThisIsAClientInADifferentRepositoryTest.kt
package com.pduda.field

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test

class ImagineThisIsAClientInADifferentRepositoryTest {
    @Test
    @Throws(Exception::class)
    fun singleItem_numberOfProductsInTheCart() {
        val client = ImagineThisIsAClientInADifferentRepository()
        Assertions.assertThat(client.formattedTotalPrice(50)).isEqualTo("Total price is 50 euro")
    }
}